docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2910,6 +2910,479 @@ have not been included. These are:
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 1%
+    - N/A
+    - Incorrect Select
+    - Noise for all types of addresses will work in the same way 
+  * - Zip Code 
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 1%
+    - 0.1 
+    - Numeric miswriting
+    - Applies to home, mailing, and employer addresses 
+  * - Relationship to head of household 
+    - Census 
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - Sex 
+    - Census, Household Surveys, WIC 
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - Race/Ethnicity 
+    - Census, WIC
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - SSN
+    - Taxes (both), SSA
+    - 1%
+    - 0.1 
+    - "Borrowed" SSN, Copy from within Household, Numeric miswriting 
+    - Note that not all types of noise apply to all observers, details below 
+  * - ITIN
+    - Taxes 1040
+    - 1%
+    - 0.1 
+    - Copy from within Household, Numeric miswriting 
+    - Note that not all types of noise apply to all observers 
+  * - Income / Wages
+    - Taxes (both)
+    - 1%
+    - 0.1 
+    - Numeric miswriting
+    - Note that wages and income are on separate tax forms and noise is applied to each separately 
+  * - Employer ID 
+    - Taxes (both)
+    - 1%
+    - 0.1 
+    - Numeric miswriting
+    - 
+  * - Employer Name 
+    - Taxes (both)
+    - 1%
+    - 0.1 
+    - OCR, typographic 
+    - 
+  * - Type of Tax Form  
+    - Taxes (both)
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - Type of SSA Event 
+    - SSA 
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - Date of SSA Event 
+    - SSA 
+    - 1%
+    - N/A
+    - Numeric miswriting, month and day swap 
+    - 
+
+The below section further describes types of noise including any code 
+available and information for implementation. **Software engineering team - please alert the research team if any of the below looks to be particularly challenging for further discussion.**
+
+**Notes on Inputs to Noise Function Parameters for OCR, Phonetic, Typographic and Numeric Miswriting** 
+
+Currently, the user will input values that are directly used in the noise functions: 
+rows eligible for errors, and noise level at the token level. These are 
+then directly plugged into noise functions. 
+
+One limitation of this strategy is that since there is a random probability of a 
+particular character/token receiving noise, not all rows that are selected to be 
+eligble for noise will actually receive noise. Another limitation is that token 
+error rate is not very intuitive for the end user. 
+
+At some point in the future, we might make this more user friendly. However, for the 
+sake of a minimum functional model, this is satisfactory. 
+
+One function has an additional parameter. The typographic function needs an input for 
+the probability that a corrupted token contains the original token. For simplicity, 
+we will use 0.5 and the user will not be able to edit this value. 
```

Comment:
I would lead towards a smaller value than 0.5 for the default, maybe 0.1 or 0.2? Not sure it will make a huge difference for PRL, but the string comparison algorithms may behave slightly differently for different values.

I think it's fine to keep this fixed for the MVM, and I know we said we'd just have two user parameters in the final version, but I think it might be good to keep the option open to allow more fine-grained user inputs like this one in the final version.

Reply:
That is helpful! I changed it to be 0.1